tests/test_scraper.py

```
@@ -1,151 +1,365 @@
 import json
```

Ruff (0.8.2)

`json` imported but unused

Remove unused import: `json`

(F401)